### PR TITLE
Add changelog for v1.17.2

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix push mirror address backend get error Address cause setting page display error (#20593) (#20901)
+  * Fix panic when an invalid oauth2 name is passed (#20820) (#20900)
+  * In PushMirrorsIterate and MirrorsIterate if limit is negative do not set it (#20837) (#20899)
+  * Double check CloneURL is acceptable (#20869) (#20892)
+  * Ensure that graceful start-up is informed of unused SSH listener (#20877) (#20888)
+  * Pad GPG Key ID with preceding zeroes (#20878) (#20885)
+  * Fix SQL Query for `SearchTeam` (#20844) (#20872)
+  * Fix the mode of custom dir to 0700 in docker-rootless (#20861) (#20867)
+  * Fix UI mis-align for PR commit history (#20845) (#20859)
+* TRANSLATION
```
Comment (delvh) [inline CHANGELOG.md]: We have a translation tag?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Rework repo buttons (#20602, #20718) (#20719)
+* BUGFIXES
+  * Ensure delete user deletes all comments (#21067) (#21068)
+  * Delete unreferenced packages when deleting a package version (#20977) (#21060)
+  * Redirect if user does not exist on admin pages (#20981) (#21059)
+  * Set uploadpack.allowFilter etc on gitea serv to enable partial clones with ssh (#20902) (#21058)
+  * Fix 500 on time in timeline API (#21052) (#21057)
+  * Fill the specified ref in webhook test payload (#20961) (#21055)
+  * Add another index for Action table on postgres (#21033) (#21054)
+  * fix broken insecureskipverify handling in rediss connection uris (#20967) (#21053)
```
Comment (jolheiser) [inline CHANGELOG.md]:
```suggestion
  * Fix broken insecureskipverify handling in redis connection uris (#20967) (#21053)
```

**Review by jolheiser — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,52 @@ This changelog goes through all the changes that have been made in each release
...
+  * Rework repo buttons (#20602, #20718) (#20719)
+* BUGFIXES
+  * Ensure delete user deletes all comments (#21067) (#21068)
+  * Delete unreferenced packages when deleting a package version (#20977) (#21060)
+  * Redirect if user does not exist on admin pages (#20981) (#21059)
+  * Set uploadpack.allowFilter etc on gitea serv to enable partial clones with ssh (#20902) (#21058)
+  * Fix 500 on time in timeline API (#21052) (#21057)
+  * Fill the specified ref in webhook test payload (#20961) (#21055)
+  * Add another index for Action table on postgres (#21033) (#21054)
+  * fix broken insecureskipverify handling in redis connection uris (#20967) (#21053)
```
Comment (delvh) [inline CHANGELOG.md]:
```suggestion
  * Fix broken insecureskipverify handling in redis connection uris (#20967) (#21053)
```

**Review by delvh — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
 
+## [1.17.2](https://github.com/go-gitea/gitea/releases/tag/v1.17.2) - 2022-09-06
+
+* ENHANCEMENTS
+  * Fix hard-coded timeout and error panic in API archive download endpoint (#20925) (#21051)
+  * Improve arc-green code theme (#21039) (#21042)
+  * Enable contenthash in filename for dynamic assets (#20813) (#20932)
+  * Don't open new page for ext wiki on same repository (#20725) (#20910)
+  * Disable doctor logging on panic (#20847) (#20898)
+  * Remove calls to load Mirrors in user.Dashboard (#20855) (#20897)
```
Comment (PeterBushra) [inline CHANGELOG.md:18]: x

File: CHANGELOG.md (hunk 1 above)
Comment (6543) [reply CHANGELOG.md]: Not that what I'm aware of

**Review by 6543 — APPROVED** [review]

**Pull request merged**